### ISSUE TITLE
Use of `u` to encode ligatures

### ISSUE THREAD

**Issue by lpugin** [rism-digital/pae-code-spec#28]
We have many incipits where `+` has been used for encoding ligatures instead of ties.
* For representing ligatures in mensural notation
* For representing ligatures with stem-less notes (`7.`)

We need to clarify if this is supported or not. If yes, tooling needs to be adjusted accordingly. We should also consider adjusting the specifications to use another sign.

(We also have cases where `+` has been used to represent slurs. This is simply wrong.)

**Comment by ahankinson** [comment]
Is anyone opposed to the use of a beam group `{}` to indicate ligatures for neume and mensural notation?

**Comment by lpugin** [comment]
Yes, I think this would be a bad idea. I would suggest instead a single letter but different from ties `_`. 'l' is probably not a great idea, but maybe `u`? We had `~` discussed [before](https://github.com/rism-digital/pae-code-spec/issues/28#issuecomment-1025605338).

**Comment by ahankinson** [comment]
I think it's a good idea! :-D

A beam is a kind of ligature, and it also uses a grouping character (of which we have only a few to choose from). I also like the fact that the start and the end are different, otherwise it's harder to detect incorrectly nested groups. And they don't blend in with the other letters. (of which there are many). 

`{AB{CD}` makes it easy to detect the error because you can tell at the second opening bracket that it's a problem, while `uABuCDu` is only detectable when you don't have a fourth `u`, and it's all the way to the end before you know that. `uABuCDyy` would be better if we went that way.

You could also compare something like:

`{AB}{CD}{EFG}` to `uAByuCDyuEFGy`. I think the former looks much better, and is probably easier to type. 

Except for a few sources in transitional periods, beams and ligatures are generally mutually exclusive in the different repertoires. This idea occurred to me as I work through the neume notation encoding draft, where it's likely going to be very common to have lots of grouped notes, so I think some sort of enclosing characters are preferable.

**Comment by lpugin** [comment]
> Except for a few sources in transitional periods, beams and ligatures are generally mutually exclusive

This is why the idea is not as great as it looks like ;-) 

Since we are trying to disambiguate PAEC, it seems like adopting the beam encoding for ligatures to be going in the wrong direction. I understand there is a similar concept of bracketing, but still, we are talking about something very different and applying to different duration values. This will cause a lot of additional checks and potential confusion.

Also, since the tie code has been used for ligature, I don't think we need to change to a container code `{` and `}` with something that is should remain a side feature. Staying with a flag-like code is enough in my opinion.

**Comment by lpugin** [comment]
PS Maybe I was not clear. I am simply suggesting to replace `+` with `u`. So `7C1D+E7F` would become `7C1DuE7F`
<img width="382" alt="image" src="https://github.com/rism-digital/pae-code-spec/assets/689412/a8a71f7d-1215-4880-8a36-9c8c6af27905">

**Comment by ahankinson** [comment]
I've changed the title to indicate the current direction of travel on this issue.

**Comment by BaMikusi** [comment]
Footnote for the Verovio rendering: while I agree that it's meaningful to treat neumes and ligatures together as regerds encoding, the rendering of neumes should IMO better be done by grouping, i.e., the noteheads belonging to the same neume should stand closer together. Using braces over the staff is not out of the question, but while that's a well-known practice for ligatures in modern editions, in plainchant studies it is (to the best of my knowledge) the grouping solution that has established itself as an informal standard.
